packages/experience-builder-types/src/constants.ts

```diff
@@ -12,6 +12,7 @@ export const OUTGOING_EVENTS = {
   NewHoveredElement: 'newHoveredElement',
   ComponentSelected: 'componentSelected',
   RegisteredComponents: 'registeredComponents',
+  DesignTokens: 'registerDesignTokens',
```

Comment:
```suggestion
  RegisterDesignTokens: 'registerDesignTokens',
```
nitpick: could you make the key the same as the value to be more clear?